app/views/admins/semesters/index.html.erb

```diff
@@ -10,6 +10,9 @@
           id: "edit-" + semester.season + "-" + semester.year %></td>
       <td><%= link_to "Delete", admin_semester_path(semester), method: :delete, class: 'button',
           id: "delete-" + semester.season + "-" + semester.year %></td>
+      <td><%= link_to "Set as Current Semester", set_current_semester_admin_settings_path(id: semester.id, setting_id: 1,
+          current_semester_id: semester.id), method: :post, class: 'button',
+          id: "set_as_current_semester-" + semester.season + "-" + semester.year %></td>
```

Comment:
It looks like in the controller method we only use `set_current_semester` from the params. Does the route still work if it's like:

    set_current_semester_admin_settings_path(current_semester_id: semester.id)

Also, we can use string interpolation for the id:

    "set_as_current_semester-#{semester.season}-#{semester.year}"

Alternatively (and a bit cleaner):

    id: dom_id(semester, :set_as_current)

(See http://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_id) This does require a change in the specs though.